index.html

```diff
@@ -259,6 +259,33 @@ <h3>let's connect</h3>
             </li>
         </ul>
       </section>
+
+      <section id="contact">
+        <div class="contact-bg-form">
+          <div class="contact-form">
+            <h1>Contact me</h1>
+            <p>
+              If you have an application you are 
+              interested in developing, a feature that
+               you need built or a project that needs
+                coding. I’d love to help with it!
+            </p>
+            <form action="/" method="post">
+              <div class="inputs">
+                <input type="text" name="firstname" placeholder="Abdullah Ahmadi">
+              </div>
+              <div class="inputs">
+                <input type="email" name="email" placeholder="abdullahahmadi019@gmail.com">
+              </div>
+              <div class="inputs-msg">
+                <textarea name="msg" id="msg" placeholder="Write your message here..."></textarea>
+              </div>
```

Comment:
- [ ] As specified in the project requirements kindly consider enhancing the form's input fields by adding the `maxlength` and `required` attributes. Incorporating these attributes will aid in validating user information, ensuring that users do not submit excessive data or leave required fields empty. This will result in a more user-friendly experience and help maintain data integrity.